src/main/java/com/lalabrand/ecommerce/security/jwt_token/JwtService.java

```diff
@@ -38,19 +44,20 @@ private SecretKey getKey() {
         return Keys.hmacShaKeyFor(decodedKey);
     }
 
-    public String generateToken(String email, String id) {
+    public String generateToken(String email, String userId, Integer passwordVersion) {
```

Comment:
Why can't you pass `UserDetailsImpl` as a parameter? You use `UserDetailsImpl` to validate the token.

Reply:
Yeah! Good case, i will fix it